docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  unpast:
+    build: .
+    environment:
+      - PYTHONPATH=/app
+    volumes:
+      - ./:/app
+    working_dir: /app
+    entrypoint:
+      - /bin/sh
+      - -c
+      - "pytest tests/test_run_unpast.py && pytest tests/utils/test_method.py"
```

Comment:
"pytest tests"?

Btw, maybe we don't need this file? Looks like it tests unpast in docker, which is now a part of docker-publish or tests workflows.

Reply:
While building the docker, we need to test the WGCNA installation because sometimes it installs incorrectly, but without any errors or warnings. I changed the dockerfile and actions, so we don't need compose, yes.